src/usr/local/www/services_dyndns_edit.php

```diff
@@ -491,6 +492,9 @@ function setVisible(service) {
 				hideInput('ttl', false);
 				break;
 			case "namecheap":
+			case "cloudflare-v6":
+			case "cloudflare":
```

Comment:
I don't understand what you mean?
This switch statement only mentions cloudflare here, not in any other cases. In fact "cloudflare" is not specifically mentioned in this file other than here and in help text.